scripts/camera.lua

```diff
@@ -294,4 +318,54 @@ function Camera.setTransitionPeriod(camera, transitionPeriod)
     Camera.updateConfig(camera)
 end
 
+--- @param player       LuaPlayer
+--- @param camera       Camera.camera   the camera this box is for
+--- @param iconSet      table           corner icons
+--- @param centerPos    table?          x,y pair giving the center of the box, nil if deleting tags
+--- @param zoom         number?         zoom factor for the box, ignored if deleting tags
+function Camera.refreshChartTags(player, camera, iconSet, centerPos, zoom)
+    -- we can't do this without a player or force
+    if not player or not player.force then
+        return
+    end
+
+
+    local chartTags = camera.chartTags
+    local function modifyTag(icon, pos)
```

Comment:
This function name is outdated, we should rename it to `createTag()` now

Reply:
Agreed. I'll commit a rename when I am next at a computer.